add more info to dialogs

ebbit1q opened the pull request:
adds descriptive strings to the register, password reset request,
password reset challenge request, password reset token dialogs
adds tip to set manager to use ctrl a to select all sets
change sizes in set manager
moves default server info to settings instead of having it hardcoded in
each dialog

default size of sets manager is now this
![image](https://user-images.githubusercontent.com/36401181/112248786-d5625700-8c56-11eb-820d-de51dd4e340e.png)

tooomm commented on cockatrice/src/dlg_manage_sets.cpp:
```diff
@@ -128,16 +129,17 @@ WndSets::WndSets(QWidget *parent) : QMainWindow(parent)
-                      tr("Only cards in enabled sets will appear in the card list of the deck editor") + "<br><br>" +
-                      "<b>" + tr("Card Art") + ":</b> " + tr("Image priority is decided in the following order") +
-                      "<ol><li>" + tr("CUSTOM Folder") +
+    labNotes->setText(tr("Use ctrl+a to select all sets in the view.") + "<br><b>" + tr("Deck Editor") + ":</b> " +
```
```suggestion
    labNotes->setText(tr("Use CTRL+A to select all sets in the view.") + "<br><b>" + tr("Deck Editor") + ":</b> " +
```

<br>

Regarding all hintsGrid:
Don't you think the extra line breaks were useful?
Because of how lists are displayed you now have two blocks, but the second block being bullet points of the last element of the first block. The connection and context is not as clear now as seen in your screen.

Yours:
![](https://user-images.githubusercontent.com/36401181/112248786-d5625700-8c56-11eb-820d-de51dd4e340e.png)

Compared to what is currently in client:
![hints](https://user-images.githubusercontent.com/9874850/112284528-a2858680-8c89-11eb-9f08-b89d6b6ccc4b.png)

ebbit1q replied:
I wanted to make it smaller, it's taking up a lot of space

tooomm replied:
I see, sure. Both not ideal. :)

ebbit1q commented:
![image](https://user-images.githubusercontent.com/36401181/112414614-9f8aa480-8d22-11eb-8efd-a8f0cb65d1d2.png)
I made the dialog hint a little smaller again

ebbit1q commented:
in order to make the list smaller I stopped disabled sets from being used for image art, this resulted in a >1% performance increase as shown with the added timer

ebbit1q commented:
I removed a few needless entries to the server dialog settings, among which a password entry that saves the password you used last in the register or recover dialog, whether you want it or not............
None of these entries are used anymore since the new login dialog was added, even though they all accounted for entries to be used by the old dialog they now do nothing.
Its possible to adjust these changes to try to mess with the new login dialog but imo it's not worth it because it doesn't in the current state.
I also fixed the remove function, which used to remove an entry but not move all other entries back by one.
